astropy/coordinates/sky_coordinate.py

```diff
@@ -1888,17 +1900,33 @@ def guess_from_table(cls, table, **coord_kwargs):
             ends_with_comp = r'.*(\W|\b|_)' + comp_name + r'\b'
             # the final regex ORs together the two patterns
             rex = re.compile('(' + starts_with_comp + ')|(' + ends_with_comp + ')',
-                             re.IGNORECASE | re.UNICODE)
-
-            for col_name in table.colnames:
-                if rex.match(col_name):
-                    if comp_name in comp_kwargs:
-                        oldname = comp_kwargs[comp_name].name
-                        msg = ('Found at least two matches for  component "{0}"'
-                               ': "{1}" and "{2}". Cannot continue with this '
-                               'ambiguity.')
-                        raise ValueError(msg.format(comp_name, oldname, col_name))
-                    comp_kwargs[comp_name] = table[col_name]
+              re.IGNORECASE | re.UNICODE)
```

Comment:
indent off?

Reply:
yes, thanks. I fixed it by all making the previous argument shorter using f-strings.